packages/flutter/test/material/tooltip_test.dart

```diff
@@ -1442,8 +1444,68 @@ void main() {
       'semantics: excluded',
       'wait duration: 0:00:01.000000',
       'show duration: 0:00:02.000000',
+      'triggerMode: TooltipTriggerMode.Manual',
+      'provideTriggerFeedback: true',
     ]);
   });
+  testWidgets('Tooltip trigger on tap when trigger mode is tap', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Tooltip(
+          message: tooltipText,
+          triggerMode: TooltipTriggerMode.Tap,
+          child: SizedBox(width: 100.0, height: 100.0),
+        ),
+      ),
+    );
+
+    final Finder tooltip = find.byType(Tooltip);
+    final TestGesture gestureTap = await tester.startGesture(tester.getCenter(tooltip));
+    await gestureTap.up();
+    await tester.pump();
+    expect(find.text(tooltipText), findsOneWidget);
+  });
+  testWidgets('Tooltip does not trigger on tap when trigger mode is longHold', (WidgetTester tester) async {
```

Comment:
We should also test vice versa, that it is not triggered on longPress when the mode is tap.

Reply:
Actually that's not possible. Due to the way the `GestureDetector` works, clicking on a widget long time and then releasing will also trigger `onTap` property.

I thought about adding a doc to the `TooltipTriggerMode.tap`, although this is behavior specific so should probably sit on `Tooltip`, but not sure where to put it, or to which extend does that behavior go. Or for that matters - if it's even a bug or the intended behavior.

Reply:
Ah! I did not realize. Thanks!